runtime/src/gpu/cuda/gpu-cuda.c

```diff
@@ -188,29 +211,125 @@ void chpl_gpu_impl_launch_kernel_flat(int ln, int32_t fn,
                               nargs, args);
 }
 
-void chpl_gpu_impl_copy_device_to_host(void* dst, void* src, size_t n) {
-  chpl_gpu_ensure_context();
+static bool chpl_gpu_allocated_on_host(const void* memAlloc) {
```

Comment:
We have `chpl_gpu_is_device_ptr` why not name this `chpl_gpu_is_host_ptr` or is there some nuance to this I'm missing?